packages/firestore/src/local/local_documents_view.ts

```diff
@@ -56,6 +57,13 @@ import { PersistencePromise } from './persistence_promise';
 import { PersistenceTransaction } from './persistence_transaction';
 import { RemoteDocumentCache } from './remote_document_cache';
 
+export class LocalDocumentsResult {
```

Comment:
Comment.

Reply:
So the Web SDK has an interface called `LocalWriteResult`. [Ref](https://cs.opensource.google/firebase-sdk/firebase-js-sdk/+/master:packages/firestore/src/local/local_store_impl.ts;l=100-103?q=localwriteresult&ss=firebase-sdk), which is the same thing I believe.

The iOS SDK has a similar thing. [Ref](https://cs.opensource.google/firebase-sdk/firebase-ios-sdk/+/master:Firestore/core/src/local/local_write_result.h;l=32?q=localwriteresult&ss=firebase-sdk).

The Web SDK so far has not used `LocalDocumentResult` so far, and the tests also use `LocalWriteResult`.

I think we should use `LocalWriteResult` in this PR.

Potentially, _if we think LocalWriteResult is the better way to go_, we should make a separate PR that swaps all usecases of `LocalWriteResult` with `LocalDocumentResult`. Alternatively, we can update Android to match Web and iOS.

thoughts?

Reply:
either way we must make sure all of them use the same name

Reply:
Changed to `LocalWriteResult`